src/app/workflow/view/view.component.html

```diff
@@ -91,16 +91,20 @@
     >
       Set as Default Version
     </button>
-    <button
-      mat-menu-item
-      color="accent"
-      [disabled]="
-        (isRefreshing$ | async) || version.frozen || (workflow?.mode === WorkflowType.ModeEnum.DOCKSTOREYML && !version.legacyVersion)
-      "
-      (click)="refreshVersion()"
-      matTooltip="Refresh an individual version. Disabled for frozen versions and GitHub App workflows."
-    >
-      Refresh Version
-    </button>
+    <div matTooltip="Refresh an individual version. Disabled for GitHub App workflows, Hosted workflows, and frozen versions.">
```

Comment:
I would not capitalize the "H" in "Hosted" -- our doc doesn't do it (besides in titles, which makes sense): https://docs.dockstore.org/en/stable/getting-started/hosted-tools-and-workflows.html?highlight=hosted#adding-a-hosted-workflow